conclave-common/src/main/kotlin/com/r3/conclave/common/internal/CallInterfaceStackFrame.kt

```diff
@@ -0,0 +1,9 @@
+package com.r3.conclave.common.internal
+
+import java.nio.ByteBuffer
+
+class CallInterfaceStackFrame<CALL_TYPE>(
```

Comment:
Can you add a KDoc for this class, since it's non-trivial what this class is involved in.